ShoppingList/src/main/java/org/openintents/shopping/ui/ShoppingActivity.java

```diff
@@ -2528,6 +2528,22 @@ protected void onPrepareDialog(int id, Dialog dialog) {
             case DIALOG_GET_FROM_MARKET:
                 DownloadOIAppDialog.onPrepareDialog(this, dialog);
                 break;
+            case DIALOG_DELETE_ITEM:
+                if (mItemsView != null && dialog instanceof AlertDialog) {
+                    ListAdapter adapter = mItemsView.getAdapter();
+                    if (adapter != null && adapter instanceof CursorAdapter) {
+                        Cursor c = (Cursor) adapter.getItem(mDeleteItemPosition);
+                        if (c != null) {
+                            String itemName = c.getString(mStringItemsITEMNAME);
+                            if (itemName != null) {
+                                ((AlertDialog) dialog).setMessage(String.format(
+                                        getResources().getString(R.string.delete_item_confirm).toString(),
```

Comment:
Please use getString(R.string.delete_item_confirm, itemName) directly, or why the String.format?

Reply:
Done